### ISSUE TITLE
Unicode UTF-8 Characters are displayed wrong when connecting over SSH from ArcaOS

### ISSUE THREAD

**Issue by explit7** [bitwiseworks/rpm-issues#15]
Hello guys. I have a very strange problem:

Licensed ArcaOS 5.0.6.7 OS. 
Locale set to en_US
Installed openssh rpm Package.
Connecting to a linux Machine works, but UTF-8 Characters are not displayed, or displayed wrong.

In the CONFIG.SYS i tried to:
SET TERMTYPE=vt220
or
SET TERMTYPE=xhost
or another available parameters, with the same problem.
See attached Midnight Commander Photo to demonstrate the problem.

Do you have some ideas?

Thanks 

Also opened an ArcaOS Suport Ticket, but they suggest me to open Github issue.
https://mantis.arcanoae.com/view.php?id=3098

![UTF(1)](https://user-images.githubusercontent.com/11852348/110216299-a609c700-7eae-11eb-9321-992df0f1b283.jpg)

**Comment by SilvanScherrer** [comment]
As this is not a rpm issue please use the right project next time. https://github.com/bitwiseworks/openssh-portable/issues in this case.
Which openssh did you install?

**Comment by explit7** [comment]
Sorry Silvan, Lewis Rosenthal pointed me to post the issue here. 
I have OpenSSH_7.7p1, OpenSSL 1.0.2u 20 Dec 2019 installed with yum from ArcaOS

By the way, i have also the same Problem mentioned here:

https://github.com/bitwiseworks/openssh-portable/issues/1

I opened klusr and pointed ssh to have it home folder under D:\ExPLIT\.ssh\
But it didn't helped.

I got the same Could not create directory '/nonexistent/.ssh'. errors and broken Unicode characters when connecting to the linux machine.

Thanks
